coininfo.js

```diff
@@ -6192,6 +6192,885 @@ const coininfo = {
+    total_supply: null,
+    circulating_supply: 32145403414.0,
```

Comment:
Please remove .0, so it is just an integer of 32145403414 
The same for other coins that follow